gallifreyan.js

```diff
@@ -19,6 +19,7 @@ var lines           = [],
 var dirtyRender     = true; //whether GUI and red dots will be drawn
 
 var deleteLineMode  = false;//whether next selected line will be deleted
+var addLineMode     = false;//next mouse click will place the first end of a new line
```

Comment:
I'd make this comment more explicit:

    // Both ends of a line move with a cursor. It looks like you're placing the first end of the line.
    // A click will disable this mode, and you'll normally control one end of the line.